content/departments/cloud/trial_mi.md

```diff
@@ -121,10 +123,10 @@ When trial expires and customer do not wish to sign the deal, instance requestor
 4. Finalisation (Cloud Team member)
 
    - checkout `<CUSTOMER>/create-instance` branch in`deploy-sourcegraph-managed` repository
+   - `cd <CUSTOMER>`
    - need to set the license on the instance (the license key should be added to the issue, unless it is default PLG licence)
      - run `mi set-license -license-key "$LICENSE_KEY"` (for PLG trials flag `--plg-default` instead of `-license-key` should be used - will use shared PLG licence key)
    - obtain customer reset link via `mi reset-customer-password --email <customer admin email>` and paste it into the GitHub issue
-   - when [giving customer access](./technical-docs/v1.1/mi1-1_creation_process.md#giving-customer-access) is done via comment in New Trial Managed Instance request issue, alert in `#cloud-notifications` should be closed.
    - (PLG triel only) add the `cloud-trial/instance-ready` label on the instance request issue. This will trigger an alert in #cloud-trial-alerts.
```

Comment:
```suggestion
   - (PLG trial only) add the `cloud-trial/instance-ready` label on the instance request issue. This will trigger an alert in #cloud-trial-alerts.
```